packages/testsuite/cypress/support/containers-utils.ts

```diff
@@ -25,6 +45,36 @@ declare global {
        * @category Containers util
        */
       startWildflyContainer(): Chainable<unknown>;
+      /**
+       * Start a Wildfly container. This typical command which needs to executed in before method in every TC.
+       *
+       * This container will have a secured management API.
+       *
+       * @category Containers util
+       *
+       * @param portOffset - optional parameter to set offset for all used ports.
+       *
+       */
+      startWildflyContainerSecured(): Chainable<unknown>;
+      /**
+       * Start a SSO container. This command which needs to executed in before method to start and configure SSO.
+       *
+       * The network mode must be host. Because to SSO is accessed from a network (Wildfly to Keycloak the IP must be used)
```

Comment:
>    * The network mode must be host. Because to SSO is accessed from a network (Wildfly to Keycloak the IP must be used)
>    * and from outside of a network (The users browser is outside of network and does't know the IP of the SSO).
>    * In configuration of OICD can be used localhost or IP but then users browser or Wildfly cannot access to SSO
>    * Solution is run both service in host mode to be available at the host machine and then can be configured the
>    * OIDC to localhost.

Can we go with something like the following?

The container network mode is set to `host` to enable communication between browser (not in container), application server (in container), and Keycloak (in container). Those need to use the same network and consistent host names.

Reply:
Description of method updated.  Thank you for suggestion.